app/assets/stylesheets/comp-vue-text-editable.scss

```diff
@@ -0,0 +1,31 @@
+.gobierto-text-editable {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  max-width: 75vw;
+
+  > :first-child {
+    flex: 1;
+    margin: 0;
+    padding: 0;
+    min-width: 20px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-family: var(--font-family);
+
+    &:focus {
+      border-radius: 4px;
```

Comment:
```suggestion
@include border-radius(4px);
```

Reply:
That was initially, like you suggest. But I went back, because this stylesheet is a component. So, in order to make it work as standalone, you'll need to add `@import "mixins"` at the top.
That's the reason I tried to get rid of such dependency in the file, only to set a property.